src/utils/__tests__/processInput.test.ts

```diff
@@ -0,0 +1,16 @@
+import { sanitizeString } from "../processInput"
+
+describe("processInput", () => {
+  describe("sanitizeString", () => {
+    it.each`
```

Comment:
Clever, maybe just a bit too clever for me to follow. This is relying on some of the weird tricks with template strings, how it's the one place you can call a function without parens in JS?

Reply:
Correct, [tagged template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). The usage for these tests is documented [in the Jest docs](https://jestjs.io/docs/en/api#2-describeeach-table-name-fn-timeout).